### PR TITLE
Added issue template and fixed a bug where ESLint config

### PR DESCRIPTION
**Pull request by hirotomoyamada**
Related #1355

## Description

Added issue template and fixed a bug where ESLint config.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: CONTRIBUTING.ja.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ cd yamada-ui
 - [Storybook](https://storybook.js.org/) コンポーネントの迅速な開発とテスト
 - [Testing Library](https://testing-library.com/) コンポーネントとフックのテスト
 - [Changeset](https://github.com/atlassian/changesets) 変更のドキュメンテーション・ログの生成、リリースの管理
+- [Next,js](https://nextjs.org/) ドキュメントサイトのフレームワーク
```
Comment (KenyaMasuko) [inline CONTRIBUTING.ja.md:36]:
```suggestion
- [Next.js](https://nextjs.org/) ドキュメントサイトのフレームワーク
```

**Review by KenyaMasuko — COMMENTED** [review]
@hirotomoyamada 
Please fix the typo!

File: CONTRIBUTING.ja.md (hunk 1 above)
Comment (hirotomoyamada) [reply CONTRIBUTING.ja.md:36]: Fixed

**Review by KenyaMasuko — APPROVED** [review]
Thanks!

**Pull request merged**